planning/planning_debug_tools/scripts/trajectory_visualizer.py

```diff
@@ -173,11 +171,10 @@ def __init__(self):
             )
             self.setPlotTrajectoryVelocity()
 
-        plt.show()
```

Comment:
@TakaHoribe 
`plt.show()` causes lock in this PR's change?

Reply:
Seems that was the cause

Reply:
Note:  thread lock problem still happens with this PR